### DISCUSSION TITLE
js error on http management page

### DISCUSSION THREAD

**Discussion by clcaod** [rabbitmq/rabbitmq-server#9621] [Other] [answered]
### Describe the bug

![image](https://github.com/rabbitmq/rabbitmq-server/assets/108561101/79e3137a-59fc-4f2b-823b-c4f25ce01e68)

### Reproduction steps

just install RabbitMQ version 3.11x,3.12x

### Expected behavior

why would that happened? another issue is that is you install use home-brew, you will get issue when you start service, only 3.12.x version had that problem

**Comment by michaelklishin** [comment, marked as answer]
@clcaod clearing your browser cache will help. As for Homebrew, it does not enable all [feature flags](https://rabbitmq.com/upgrade.html) on upgrades, which [3.12.0 requires](https://rabbitmq.com/upgrade.html).

You can enable them explicitly or, since Homebrew installations are non-production in practice, remove node's data directory and let it start, then enable all feature flags.
